src/calculateRentalCost.js

```diff
@@ -27,8 +27,24 @@
  * @return {number}
  */
 
+const STANDARD_PRICE = 40;
+const SHORT_TERM = 3;
+const MID_TERM_DISCOUNT = 20;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  if (days < SHORT_TERM) {
+    return days * STANDARD_PRICE;
+  }
+
+  if (days >= SHORT_TERM && days < LONG_TERM) {
+    return days * STANDARD_PRICE - MID_TERM_DISCOUNT;
```

Comment:
Even better, why don't you create a const for standardTerm or something that will be responsible for  this calculation? 

`days * STANDARD_PRICE`